samples/notebooks/iterative_phase_estimation.ipynb

```diff
@@ -0,0 +1,391 @@
+       "\n",
+       "## Connect to Azure Quantum and set up target\n",
+       "\n",
+       "First, we must configure the qsharp module to connect the azure workspace and specify a target. The Quantinuum H1 emulator target `quantinuum.sim.h1-1e` is chosen by default, which performs hardware-modeled noisy simulation. Running the experiments in this notebook against that target will consume approximately 25 HQC.\n",
+       "\n",
+       "We also configure the target profile as `Adaptive_RI` to indicate we use the QIR profile with support for mid-circuit measurement, measurement-based control flow, and classical integer computation as part of compilation.\n",
+       "\n",
+       "Replace the `subscription_id`, `resource_group`, `name`, and `location` connection parameters with the values for your configured Azure Quantum Workspace, or try copying the Python code for connecting to your workspace from the VS Code Azure Quantum Extension."
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "import qsharp\n",
+       "\n",
+       "qsharp.init(target_profile=qsharp.TargetProfile.Adaptive_RI)"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "import azure.quantum\n",
+       "\n",
+       "workspace = azure.quantum.Workspace(\n",
+       "    subscription_id = \"SUBSCRIPTION ID\",\n",
+       "    resource_group = \"RESOURCE GROUP\",\n",
+       "    name = \"WORKSPACE NAME\",\n",
+       "    location = \"LOCATION\",\n",
+       ")"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "target = workspace.get_targets(\"quantinuum.sim.h1-1e\")"
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "## Two Dimensional Inner Product Calculation Using Iterative Phase Estimation on Three Qubits\n",
+       "\n",
+       "This notebook demonstrates an iterative phase estimation within Q#. It will use iterative phase estimation to calculate an inner product between two 2-dimensional vectors encoded on a target qubit and an ancilla qubit. An additional control qubit is also initialised which will be the only qubit used for measurement.\n",
+       "\n",
+       "The circuit begins by encoding the pair of vectors on the target qubit and the ancilla qubit. It then applies an Oracle operator to the entire register, controlled off the control qubit (which is set up in the $\\\\ket +$ state). The controlled Oracle operator generates a phase on the $\\\\ket 1$ state of the control qubit. This can then be read by applying a H gate to the control qubit to make the phase observable when measuring."
```

Comment:
The kets in this cell weren't rendering properly for me, and removing the extra slash seems to have done the trick:
```suggestion
       "The circuit begins by encoding the pair of vectors on the target qubit and the ancilla qubit. It then applies an Oracle operator to the entire register, controlled off the control qubit (which is set up in the $\\ket +$ state). The controlled Oracle operator generates a phase on the $\\ket 1$ state of the control qubit. This can then be read by applying a H gate to the control qubit to make the phase observable when measuring."
```